### DISCUSSION TITLE
找不到PDFPatcher.exe文件

### DISCUSSION THREAD

**Discussion by Learning356** [wmjordan/PDFPatcher#34] [问答]
只找到了7za.exe，7za.exe打开后没有反应？

**Comment by Yilun-D** [comment]
我也是下载了以后找不到PDFPatcher.exe，按照作者的介绍，PDFPatcher.exe 应该在 bin 目录下，但是并没有 bin目录。。。

**Reply by calcitem** [reply]
不编译怎么会有可执行文件输出呢？

编译产物也不可能纳入版本控制嘛。

要下载现成的可执行文件到网盘。
